pages/index.vue

```diff
@@ -48,7 +48,46 @@
         </div>
 
         <i18n-page-wrapper>
-            <div class="sponsor-section">
+            <div class="info-section section-wrapper">
```

Comment:
The mobile layout does not conform to the [design spec](https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=31%3A1821). Please check again.
<img width="501" alt="image" src="https://user-images.githubusercontent.com/24987826/165798403-ec4ac919-9872-46f9-b1bf-f61b8c82d824.png">